### DISCUSSION TITLE
Moving Obstacle Avoidance

### DISCUSSION THREAD

**Discussion by oislos** [do-mpc/do-mpc#288] [General]
Thank you for sharing wonderful toolbox!

I'm currently trying to implement a dynamic system. It is like an extension of the DIP, but I wonder if I can set an obstacle to move with time and make controller to avoid the obstacle. My question is how to update obstacle position with time in model or controller?

**Comment by yixjia** [comment]
Hi @oislos , 
I'm also trying to use do-mpc for dynamic system with moving obstacles. Have you figured out how to update the obstacle's position?
Thanks!
